.github/CODEOWNERS

```
@@ -15,29 +18,26 @@ serverless/messaging/ @RoRoJ
 # Benedikt
 compute/gpu/ @bene2k1
 compute/instances/ @bene2k1
```

```suggestion
compute/ @bene2k1
```

You have put just `bare-metal/` - maybe we can do the same for compute as both products in this category are mine? as you wish :)